guidelines/enhancement_template.md

```diff
@@ -272,6 +272,40 @@ to implement the design.  For instance,
  > 1. This requires exposing previously private resources which contain sensitive
   information.  Can we do this?
 
+## Security
+
+This section should cover the security impact of the change, if any. For example,
+does it introduce new privileges or capabilities? Does it expose new risks?
+
+### Credentials and RBAC
+
+What credentials will the new component be using if it needs to talk to either the control plane or other components? What RBAC needs to be applied?
+
+Consider the following:
+- Which service accounts will be affected by the change, what permissions will the feature code need?
+- Will end users need to make any security considerations when enabling this feature?
+- What is the scope of the required RBAC?
+  - Are cluster wider permissions required?
```

Comment:
If this is a thing we want to control or know, we should add an e2e test/registry similar to the certificate ownerships that requires a high level ACK.  The list can determine if the source CVO, any serviceaccount from a namespace created by CVO or created by a serviceaccount in a namespace created by the CVO (etc).

Asking without enforcement is hollow and this one is easy.

Reply:
Neat idea, I wonder how many would fail this test right now.

Off the top of my head, and in my own areas, I know both CCMs and the operator would, the CAPI operator would (it touches the MAPI namespace). I suspect many have reasons so perhaps this isn't such a strong argument